hnn_core/network.py

```diff
@@ -352,6 +353,9 @@ def __init__(self, params, add_drives_from_params=False, legacy_mode=True):
 
         # XXX this can be removed once tests are made independent of HNN GUI
         # creates nc_dict-entries for ALL cell types
+        if legacy_mode:
```

Comment:
So I suppose we need to update the "gold standard" dipole waveform now for the tests to pass?

Reply:
that's the least of our concerns at the moment 🤣 see the last commit gives this error:

```py
kwargs = {'location': 'proximal', 'sigma': 0.0, 'tstart': 1000.0, 'tstop': 170}

    def _check_drive_parameter_values(drive_type, **kwargs):
        if 'tstop' in kwargs:
            if kwargs['tstop'] is not None:
                if kwargs['tstop'] < 0.:
                    raise ValueError(f'End time of {drive_type} drive cannot be '
                                     'negative')
                if 'tstart' in kwargs and kwargs['tstop'] < kwargs['tstart']:
>                   raise ValueError(f'Duration of {drive_type} drive cannot be '
                                     'negative')
E                   ValueError: Duration of bursty drive cannot be negative

hnn_core/drives.py:80: ValueError
```

basically the question is how can we make `add_drives_from_params=True` work with `legacy_mode=False`? That's what we need for the new GUI ...